etc/ProdOfflineConfiguration.py

```diff
@@ -526,6 +526,21 @@
                sizePerEvent=155,
                scenario=cosmicsScenario)
 
+DATASETS = ["CosmicsHLTMonitor"]
+
+for dataset in DATASETS:
+    addDataset(tier0Config, dataset,
```

Comment:
sorry @LinaresToine it just occurred to me, but `HLTMonitor` is an expressly reconstructed PD, see

https://github.com/dmwm/T0/blob/a65498263e78e20fc8eced63e2e86acb1c09d7c5/etc/ProdOfflineConfiguration.py#L310

I think it should be the same for `CosmicsHLTMonitor`